setup.cfg

```diff
@@ -1,3 +1,7 @@
 [pycodestyle]
 ignore = E121, E126, E402, W504
 max-line-length = 100
+
+[pydocstyle]
+inherit = false
+ignore = D105, D203, D212, D213, D403, D404, D406, D407
```

Comment:
I'm just curious why is this needed? Do we need different behaviour than other repos?